src/rendering/cubemap.h

```diff
@@ -54,7 +54,7 @@ i32 Cubemaps_Find(const Cubemaps* maps, Guid name);
 void Cubemap_New(Cubemap* cm, i32 size);
 void Cubemap_Del(Cubemap* cm);
 
-Cubeface VEC_CALL Cubemap_CalcUv(float4 dir, float2* uvOut);
+pim_inline Cubeface VEC_CALL Cubemap_CalcUv(float4 dir, float2* uvOut);
```

Comment:
Got a few of these where the definition was inline but not the declaration, I guess clang and gcc are more strict about this.

Reply:
Probably from adding 'static' which changes the linkage